skema/gromet/execution_engine/execution_engine.py

```diff
@@ -0,0 +1,198 @@
+import yaml
+import argparse
+import asyncio
+from pathlib import Path
+from typing import Any, List, Dict
+
+import torch
+from gqlalchemy import Memgraph
```

Comment:
I think this is missing from pyproject.toml

Reply:
Ah yes you are right, do you think it should be added as a [dev] dependency, or as something else?

Reply:
I was thinking just under our list of project dependencies (which is where `tree-sitter` is currently): https://github.com/ml4ai/skema/blob/d3bc7d19fd87309c84c12c0961d5a43977a2f963/pyproject.toml#L19